### PR TITLE
feat: Add last_launched_time to images

### LINKED ISSUE #552: AMI Endpoint Has new attribute
### Describe the resource.

AMI Resource has been updated with new attribute
https://aws.amazon.com/about-aws/whats-new/2022/03/amazon-ec2-ami-timestamp-instance-launch/

### Use Case

monitoring AMI usage

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by irmatov**
Closes #552

### CODE REVIEW

File: resources/services/ec2/images.go [hunk 1]
```diff
@@ -305,3 +311,28 @@ func fetchEc2ImageBlockDeviceMappings(ctx context.Context, meta schema.ClientMet
...
+	cl := meta.(*client.Client)
+	image := resource.Item.(types.Image)
+	svc := cl.Services().EC2
+	opts := ec2.DescribeImageAttributeInput{
+		Attribute: types.ImageAttributeNameLastLaunchedTime,
+		ImageId:   image.ImageId,
+	}
+	result, err := svc.DescribeImageAttribute(ctx, &opts, func(options *ec2.Options) {
+		options.Region = cl.Region
+		options.EndpointResolver = ec2.EndpointResolverFromURL(fmt.Sprintf("https://ec2.%s.amazonaws.com", cl.Region))
```
Comment (bbernays) [inline resources/services/ec2/images.go:325]: Can we document why this is needed?

File: resources/services/ec2/images.go [hunk 2]
```diff
@@ -305,3 +311,28 @@ func fetchEc2ImageBlockDeviceMappings(ctx context.Context, meta schema.ClientMet
...
+	})
+	if err != nil {
+		return nil
```
Comment (bbernays) [inline resources/services/ec2/images.go:328]: Is there a reason why we are not wrapping this error?

**Review by bbernays — CHANGES_REQUESTED** [review]

File: resources/services/ec2/images.go (hunk 2 above)
Comment (irmatov) [reply resources/services/ec2/images.go:328]: Good catch, I still haven't used to do it. BTW return value is wrong here :D

File: resources/services/ec2/images.go (hunk 1 above)
Comment (irmatov) [reply resources/services/ec2/images.go:325]: Hm, I am not really sure, just kinda copied from the function above.

**Review by bbernays — APPROVED** [review]
LGTM

**Review by roneli — APPROVED** [review]
LGTM

**Pull request merged**